upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -1006,13 +996,6 @@ func setupNodes(opt *NewClusterOptions, cluster *api.Cluster, zoneToSubnetMap ma
 			g.Spec.Zones = []string{zone}
 		}
 
-		if cloudProvider == api.CloudProviderAWS {
```

Comment:
I think we need to leave this in for k8s < 1.27

Reply:
It is the default and has been moved to model.

Reply:
The only thing in model I see is the new default for k8s >= 1.27. When creating new clusters with k8s < 1.27 we should still have the behavior of explicitly setting these fields.

Reply:
Model has defaults and those take care of keeping the old behaviour:
```go
	lt := &awstasks.LaunchTemplate{
		HTTPPutResponseHopLimit:      fi.PtrTo(int64(1)),
		HTTPTokens:                   fi.PtrTo(ec2.LaunchTemplateHttpTokensStateRequired),
	}
```

Reply:
New clusters with k8s < 1.27 created by kOps >= 1.27 should have IMDSv2 hop limit 1, just like new clusters with k8s < 1.27 created by kOps 1.26.

Reply:
They do, just diferent approach.

Reply:
I can now see how we were previously defaulting the hop limit to 1. But it looks like there's an issue with requiring tokens in new clusters with k8s < 1.27.

New clusters with k8s < 1.27 created by kOps >= 1.27 should have tokens required, just like new clusters with k8s < 1.27 created by kOps 1.26.

Reply:
I would like ALL clusters to have tokens required. It is a change in default behaviour, but it's something that we pushed for all new clusters since kOps 1.22. It's been a long time to accept this behaviour. Operators should move to required tokens sooner rather than later.